src/patternfly/components/ActionList/action-list.scss

```
@@ -1,29 +1,38 @@
 // @debug $action-list; // check your variable names located in src/patternfly/sass-utilities/component-namespaces.scss
 
-.#{$action-list} {
-  --#{$action-list}--m-icon--spacer: 0;
-  --#{$action-list}--child--spacer-base: var(--#{$pf-global}--spacer--md);
-  --#{$action-list}--group--spacer-base: var(--#{$pf-global}--spacer--2xl);
+:where(:root),
+:where(.#{$action-list}) {
+  --#{$action-list}--RowGap: var(--pf-t--global--spacer--sm);
+  --#{$action-list}--ColumnGap: var(--pf-t--global--spacer--sm);
```

per the design, looks like this should be a medium spacer

```suggestion
  --#{$action-list}--ColumnGap: var(--pf-t--global--spacer--md);
```